rules/xcodeproj.bzl

```diff
@@ -544,14 +545,24 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         target_settings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = " ".join(
             ["\"%s\"" % d for d in defines_without_equal_sign],
         )
+
+        # Passing framework search paths via -F flag so
+        # LLDB has the necessary information to create the ASTs
+        #
+        # Fixes an issue where the debugger output was empty in Xcode
+        # when debugging .swift files
+        #
+        # Writing to a file and loading this in 'tools/xcodeproj_shims/installers/lldb-settings.sh'
+        # because for big projects Xcode will complain about the size of 'BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE'
+        lldb_framework_search_paths_filename = "%s_BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE" % (target_name)
+        lldb_framework_search_paths_file = ctx.actions.declare_file(lldb_framework_search_paths_filename)
```

Comment:
Does this work when debugging on a device? I don't think you're able to give it both ARM and x86_64 search paths in this way.

Reply:
Good call, I'll test that now

Reply:
We probably have a similar issue right now (even without this change) where autocomplete might not work if you build for device, but not for simulator, because the framework search paths may be pointing to the simulator search paths.

Reply:
Ah yeah makes sense to me. Overall, I'd be curious if adding `apple_framework` in XCHammer gets if device builds working. There's also new build system, extensions, device testing, and other issues it may unlock once `apple_framework` is up and running.

Reply:
Confirmed that building for device has the same behaviour before and after this PR. Build succeeds with an error tip in Xcode saying
```
Could not find module 'X' for target 'arm64-apple-ios'; found: x86_64
```
So I won't fix this as part of this PR

Reply:
Ok so it was broken before because it gave it the search paths for a simulator. I'd suggest confirming this issue won't happen if you use the right system dir, I wouldn't be surprised that it isn't hitting the error because it first fails on the system module but haven't tested it.